app/models/discussion_reader.rb

```diff
@@ -51,10 +51,6 @@ def first_read?
     last_read_at.blank?
   end
 
-  def user_or_logged_out_user
-    user || LoggedOutUser.new
-  end
-
```

Comment:
Can't quite fathom why this was here, and I don't see references to it anywhere else.